### DISCUSSION TITLE
can the screen of the controlled end be turned off?

### DISCUSSION THREAD

**Discussion by fengyuclub** [rustdesk/rustdesk#535] [General]
If the interactive interface is not needed, can the screen of the controlled end be turned off?

**Comment by JensMertelmeyer** [comment]
I think better than just displaying a black screen, the remote end should be able to display a bitmap like "Maintenance. We are working on it. Last update: 10:15". Something like that.

Also, the target machine should always have the "upper hand" and get out of it, for example by pressing an [x] symbol or by [Esc] key.

**Comment by rustdesk** [comment]
Will be ready soon, https://github.com/rustdesk/rustdesk/pull/444

**Reply by gaetanolip** [reply]
and in case the man in front of  the host machine touched the the mouse or keyboard i would like the log in screen appeared :))))

**Reply by TtuxX** [reply]
Hi there @rustdesk, is there a way to have it work when controlling an Android device from a computer ? I mean have the screen on Android turned off (at least in appearance) ? The use case for me is to control my phone while saving battery. Thank you very much ! :)
